### ISSUE TITLE
Training datasets.

### ISSUE THREAD

**Issue by FYtrace** [jinlinyi/PerspectiveFields#2]
Thank you for selflessly releasing the code of such fantastic work. I got some trouble producing training sets. Will you release the training datasets or generation code?

**Comment by jinlinyi** [comment]
Hi, thank you for your interest in our project. 

We purchased the training panoramas from [360cities](https://www.360cities.net/). You will have to obtain the images from the website. 

Alternatively, you can create your own dataset from panoramas or images with ground truth camera parameters. Please refer to this [jupyter notebook](https://github.com/jinlinyi/PerspectiveFields/blob/main/jupyter-notebooks/camera2perspective.ipynb) for how to crop a panorama and get the corresponding Perspective Fields from parameters. 

We will release our test set on Stanford2D3D and TartanAir soon.

**Comment by FYtrace** [comment]
Thanks for your quick and details reply ~
